process/incubation.md

```diff
@@ -126,9 +131,13 @@ Changes to the Swift Server Ecosystem index page will be announced by the SSWG u
 * Include list of adopters for at least the primary repo ideally laid out in an ADOPTERS.md files or logos on the project website
 * Optionally, have a [Developer Certificate of Origin](https://developercertificate.org) or a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement)
 
-## Process Diagram
+## Security Best Practices
 
-![process diagram](incubation.png)
+Project authors that discover, or been adviced of volnerabilities in their libraries, must report them to the SSWG within 10d. Graduated libraries are expected to address volnerabilities within 30d.
+
+The SSWG will publicly share a list of volnerabilities and fixes on it's public website [TBD] to inform the Swift Server user community.
```

Comment:
ideas on how to publish these? markdown file in this repo?

Reply:
I think we also need something that 'pushes' to interested folks where people can subscribe. Maybe a good old mailing list that then links to the markdowns published here?

Reply:
Hmm, good point; With Akka we did: 
- https://groups.google.com/forum/#!forum/akka-security for push
- pointing to https://doc.akka.io/docs/akka-http/current/security.html -> https://doc.akka.io/docs/akka-http/current/security/2018-09-05-denial-of-service-via-decodeRequest.html
- and CVEs also pointing to the same specific page about an CVE https://www.cvedetails.com/vulnerability-list/vendor_id-16757/Akka.html

Reply:
@weissi we could create a forum category as @ktoso describes, with links back here? then folks can subscribe on the category?

Reply:
General forums category sounds quite good actually. Going to a google mailing group would be a bit weird; and the forum is quite discoverable as well since we do all our other stuff there 👍

Reply:
> It's not "really push"

you could watch a forum category which will then email you, which is almost the same?

Reply:
I've found the watching categories be a bit flakey with the emails in discourse. To actually enforce getting emails for every new thread one has to use the mailing list mode I believe... but that's a forums global setting for a given user. (looked into this a lot when we moved to discuss.akka.io )